### ISSUE TITLE
Unable to start on Linux

### ISSUE THREAD

**Issue by AgentScrubbles** [BinaryAlley/DatasetTag#1]
Unable to start the app on Linux, specifically PopOS (based on Ubuntu).

```
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at DatasetTag.MainWindow.PopulateDefaultCategories() in /home/<<user>>/src/DatasetTag/src/DatasetTag/MainWindow.axaml.cs:line 427
   at DatasetTag.MainWindow.MainWindow_Opened(Object sender, EventArgs e) in /home/<<user>>/src/DatasetTag/src/DatasetTag/MainWindow.axaml.cs:line 1335
   at Avalonia.Controls.TopLevel.OnOpened(EventArgs e) in /_/src/Avalonia.Controls/TopLevel.cs:line 494
   at Avalonia.Controls.WindowBase.OnOpened(EventArgs e) in /_/src/Avalonia.Controls/WindowBase.cs:line 199
   at Avalonia.Controls.Window.ShowCore(Window parent) in /_/src/Avalonia.Controls/Window.cs:line 675
   at Avalonia.Controls.Window.Show() in /_/src/Avalonia.Controls/Window.cs:line 599
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.ShowMainWindow() in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 129
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[] args) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 118
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime[T](T builder, String[] args, ShutdownMode shutdownMode) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 209
   at DatasetTag.Program.Main(String[] args) in /home/<<user>>/src/DatasetTag/src/DatasetTag/Program.cs:line 12
Aborted (core dumped)
```

If I were to guess from my own development on dotnet for linux I would _assume_ this is something like Linux doesn't give the window resolution like Windows does so you have to set some defaults, but that is just a wild shot in the dark.

**Comment by BinaryAlley** [comment]
Hello, thanks for reporting the issue!
Having a look at the code line where the exception is thrown, the program tries to access a deserialized object from the configuration file, `appsettings.json`. I just tried downloading the release archive on a Debian system, extracted the files, ran the application, all went fine. Then, I closed it, removed the `appsettings.json` file from the application's directory, and tried to run it again. There was an exception, and the stack trace was identical to yours. Can you verify that the configuration file is indeed present? And if so, can you check if it's not something wrong with its permissions or something?

**Comment by AgentScrubbles** [comment]
I can confirm the `appsettings.json` is there in the src directory, but it is not being copied on build.  I tried copying it to the `bin/Debug/net7.0` directory manually where DatasetTag.dll lives, but it still gives the same exception.

I did try the release version you have, extracted and ran `dotnet DatasetTag.dll` and it ran perfectly, so it looks like it's something to do with the build, and you're probably right with it not copying the appsettings to the right place.  For me I just did `dotnet build` in the project's directory
